examples/worlds/camera_video_record_dbl_pendulum.sdf

```diff
@@ -0,0 +1,392 @@
+<?xml version="1.0" ?>
+<!--
+  Record a video from a camera sensor.
+
+  Call the following service to start video recording:
+
+      ign service -s /camera/record_video --reqtype ignition.msgs.VideoRecord --reptype ignition.msgs.Boolean --timeout 300 --req 'start: true, format:"mp4", save_filename:"test.mp4"'
+
+  Call the following service to stop video recording:
+
+      ign service -s /camera/record_video --reqtype ignition.msgs.VideoRecord --reptype ignition.msgs.Boolean --timeout 300 --req 'stop: true'
+
+  The video is saved to test.mp4
+
+-->
+
+<sdf version="1.6">
+  <world name="camera_video_record_pendulum">
+
+    <physics name="1ms" type="ignored">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+    <plugin
+      filename="libignition-gazebo-physics-system.so"
+      name="ignition::gazebo::systems::Physics">
+    </plugin>
+    <plugin
+      filename="libignition-gazebo-user-commands-system.so"
+      name="ignition::gazebo::systems::UserCommands">
+    </plugin>
+    <plugin
+      filename="libignition-gazebo-scene-broadcaster-system.so"
+      name="ignition::gazebo::systems::SceneBroadcaster">
+    </plugin>
+    <plugin
+      filename="libignition-gazebo-sensors-system.so"
+      name="ignition::gazebo::systems::Sensors">
+      <render_engine>ogre2</render_engine>
+    </plugin>
+
+    <gui fullscreen="0">
```

Comment:
I think you can remove the `<gui>` tags now that #317 is in.

Reply:
removed <gui>. d90f78e